BridgeCareApp/BridgeCareCodeFirst/Models/DeficientData.cs

```diff
@@ -0,0 +1,125 @@
+﻿using BridgeCareCodeFirst.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace BridgeCare.Models
+{
+    public class DeficientData
+    {
+        public DeficientOrTargetTab GetDeficient(SimulationResult data, int[] totalYears)
```

Comment:
Create separate models for deficient and target model data.

Reply:
Created separated models